spinalcordtoolbox/utils.py

```diff
@@ -198,6 +198,7 @@ class Metavar(Enum):
     int = "<int>"
     list = "<list>"
     float = "<float>"
+    flag = "<flag>"
```

Comment:
i would not display anything. Some people will wonder what is "flag". Just leave it blank.

Reply:
Understood! Changes made in commit [6e9e8b0](https://github.com/neuropoly/spinalcordtoolbox/commit/6e9e8b022f2b115c78de0fa531aa60d85d98461e)